crates/swc_ecma_minifier/src/compress/optimize/inline.rs

```diff
@@ -365,8 +366,21 @@ where
     }
 
     /// Check if the body of a function is simple enough to inline.
-    fn is_fn_body_simple_enough_to_inline(&self, body: &BlockStmt, param_count: usize) -> bool {
-        let cost_limit = 3 + param_count * 2;
+    fn is_fn_body_simple_enough_to_inline(
+        &self,
+        body: &BlockStmt,
+        param_count: usize,
+        usage: &VarUsageInfo,
+    ) -> bool {
+        let param_cost = param_count * 2;
```

Comment:
I'm still wondering where does this 2 come from

Reply:
It was due to a bug.
I think we can remove it at now